lib/modules/datasource/github-releases/index.spec.ts

```diff
@@ -8,25 +7,19 @@ jest.mock('../../../util/host-rules');
 const hostRules: any = _hostRules;
 
 const githubApiHost = 'https://api.github.com';
-const githubEnterpriseApiHost = 'https://git.enterprise.com';
 
-const responseBody = [
-  { tag_name: 'a', published_at: '2020-03-09T13:00:00Z' },
-  { tag_name: 'v', published_at: '2020-03-09T12:00:00Z' },
-  { tag_name: '1.0.0', published_at: '2020-03-09T11:00:00Z' },
-  { tag_name: 'v1.1.0', draft: false, published_at: '2020-03-09T10:00:00Z' },
-  { tag_name: '1.2.0', draft: true, published_at: '2020-03-09T10:00:00Z' },
-  {
-    tag_name: '2.0.0',
-    published_at: '2020-04-09T10:00:00Z',
-    prerelease: true,
-  },
-];
+const cacheGetItems = jest.fn();
+jest.mock('./cache', () => {
+  return {
+    CacheableGithubReleases: jest.fn().mockImplementation(() => {
+      return { getItems: () => cacheGetItems() };
+    }),
+  };
+});
 
 describe('modules/datasource/github-releases/index', () => {
-  const githubReleases = new GithubReleasesDatasource();
-
   beforeEach(() => {
+    jest.resetAllMocks();
```

Comment:
isn't a clear enough?

Reply:
Can lead to errors in tests that are hard to spot, isn't it?

Reply:
those are bad tests 🙃  I know we've a lot of those.

Reply:
Debugging tests that relied on the interference with previous ones' unused mocks isn't fun . That's why I always prefer to reset.